code/modules/modular_computers/file_system/programs/research/ai_restorer.dm

```diff
@@ -11,6 +11,7 @@
 	nanomodule_path = /datum/nano_module/program/computer_aidiag/
 	var/restoring = 0
 	color = LIGHT_COLOR_PURPLE
+	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP
```

Comment:
I am not sure about that.
Its usually not needed on a laptop and there are AI Diag Programs on the stationary PCs.

Imho it would make it too easy to go into the AI core with a card and modify the AIs laws to your will or "quickly verify the integrity" by carding it. (Would give more reason to the AI Rouge meta.

Reply:
This was to mostly make it not available on tablets, as it's useless on them, as AI card slot is only available on laptops and up.